docs/_content/overige/technisch/zds-vs-zgw.md

```diff
@@ -7,27 +7,22 @@ De Zaak- en Documentservices (ZDS) wijken af van de Zaakgericht werken (ZGW)
-3. De ZGW APIs kunnen alles wat ZDS ook kan.
-4. De ZGW APIs zijn opgedeeld. Er is niet meer 1 specificatie of endpoint om
-   alle operaties, zoals deze in ZDS bestonden, uit te voeren of data op te
-   vragen.
-5. De ZGW APIs zullen sneller opvolgende versies krijgen. Hierbij is het niet
-   ondenkbaar dat de Zaken API op versie 2 zit, terwijl de Besluiten API al op
-   versie 4 zit. Het is daarom ook niet zo dat ZGW een versie nummer krijgt.
-   Immers is ZGW alleen maar de verzamelnaam van alle Zaakgericht werken APIs. 
-6. De ZGW APIs worden aangetoond middels een referentie implementatie om de
-   theoretische specificaties te toetsen aan de praktijk, migratie problemen
-   te detecteren en om het vereiste gedrag te definieren daar waar de 
-   specificatie ruimte laat voor interpretaties.
-7. De ZGW APIs zijn ontwikkeld vanuit developer-first oogpunt waardoor
-   koppelingen sneller gemaakt kunnen worden.
-8. De ZGW APIs kennen alleen synchrone operaties.
-9. De ZGW APIs worden net als ZDS beheerd en onderhouden door VNG, zij het in
-   een andere vorm.
+[Zaak-Documentservices](https://www.gemmaonline.nl/index.php/Zaak-_en_Documentservices) | [ZGW APIs](https://ref.tst.vng.cloud/standaard/index)\*
+--- | ---
+SOAP/XML gebaseerd	                                                                        | REST/JSON gebaseerd
+Gebaseerd op het informatiemodel RGBZ1	                                                    | Gebaseerd op informatiemodel RGBZ2 waar van toepassing
+Verdere invulling/uitwerking van de basisberichten uit StUF ZKN	                            | Aparte calls voor aparte resources
+Gebaseerd op uitwisselen van informatie, gestructureerd volgens het informatiemodel [RGBZ 1](https://www.gemmaonline.nl/index.php/Informatiemodel_Zaken_(RGBZ))	| Gebaseerd op operaties op resources die leven in het referentiecomponent
+Ondersteunen van de functies van een zaaksysteem zoals beschreven in [GEMMA 1](https://www.gemmaonline.nl/index.php/Overzicht_GEMMA_1_referentiecomponenten)	            | Ondersteunen van functies van de referentiecomponenten zoals beschreven in [GEMMA 2](https://www.gemmaonline.nl/index.php/Overzicht_generieke_referentiecomponenten_GEMMA_2)
+Kent alleen de functionele acties die uitgewerkt zijn in de standaard.	                    | Alle functionele acties die mogelijk zijn met de CRUD functionaliteit zijn mogelijk. De ZGW API's zijn microservices die gecombineerd kunnen worden om een functionele behoefte in te vullen.
+Betrokkenen en/of gerelateerde objecten kunnen geheel of gedeeltelijk, in ieder geval met matchinggegevens doorgegeven worden. Deze worden veelal in het zaaksysteem opgeslagen.	                                                             | een relatie naar een andere resource is een verwijzing naar die resource. Die resource wordt opgeslagen in de betreffende registratie. Dat kan een zaakregistratie (ZRC) zijn maar ook een andere (met een eigen API). 
```

Comment:
Dit is wel een goede om op te merken. Het kopje "Relaties naar andere Objecttypen/Resources" zou er zo boven kunnen als je dat documenteer formaat aanhoudt.